.editorconfig

```diff
@@ -0,0 +1,126 @@
+# http://EditorConfig.org
+# https://gds-way.cloudapps.digital/manuals/programming-languages/editorconfig
+# Copy into project root and rename to .editorconfig
+
+# uncomment if top-most EditorConfig file
+# root = true
+
+[*.java]
```

Comment:
I don't think we need a configuration for .java files.

Same comments apply also for some of the other extensions like .py, erb,etc.

Reply:
Removed